Userland/Shell/CMakeLists.txt

```diff
@@ -25,9 +25,3 @@ set(SOURCES
 
 serenity_bin(Shell)
 target_link_libraries(Shell LibShell LibMain)
-
-install(DIRECTORY Tests/ DESTINATION usr/Tests/Shell
```

Comment:
I think the "proper" way to remove these tests is to ignore them via the test runner config file? 

See: [Base/home/anon/.config/Tests.ini](https://github.com/SerenityOS/serenity/blob/master/Base/home/anon/.config/Tests.ini)

Reply:
This way we don't forget they exist, they are just reported as "skipped".

I think either doing that, or deleting them is the more appropriate way to stop running them.
Hiding them by not installing them, but keeping them around leaves them in a weird limbo state IMHO.

Reply:
Thanks, that sounds a lot better!